### ISSUE TITLE
Inject Mode: [Instant] breaks GM_addStyle, setTimeout

### ISSUE THREAD

**Issue by raszpl** [Tampermonkey/tampermonkey#1524]
## Expected Behavior

red text, alert 

## Actual Behavior

nothing

## Specifications

- Chromium: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/100.0.4896.147 Safari/537.36
- TM: [v4.18.6163]
- OS: win10

## Script

```js
// ==UserScript==
// @name         New Userscript
// @version      0.1
// @description  try to take over the world!
// @match        http://example.com/
// @grant        GM_addStyle
// @run-at       document-start
// ==/UserScript==

let css = 'h1 { color:red!important;}';
GM_addStyle(css)
setTimeout(function(){alert('setTimeout works fine')}, 1000)
```

**Comment by derjanb** [comment]
Ah, instant mode. Thanks for the clarification. Will look into it. 👍

**Comment by derjanb** [comment]
Should be fixed at 4.18.6164 (in review|[crx](https://tampermonkey.net/crx/tampermonkey_beta_current.crx)|[xpi](https://firefox.tampermonkey.net/firefox-current-beta.xpi))

Chrome/Edge users, please download the crx file linked above and drag and drop it to the extensions page `chrome://extensions`.
Firefox users please install the [BETA version](https://www.tampermonkey.net/index.php?browser=firefox) or check for BETA version updates at `about:addons` 

For a quick fix please export your settings and scripts as zip or (JSON) file at the "Utilities" tab and import it back at the fixed BETA version.
